config/webpack/loaders.js

```diff
@@ -60,6 +60,7 @@ module.exports = [
         loader: 'postcss-loader',
         options: {
           sourceMap: true,
+          plugins: () => [require('autoprefixer')]
```

Comment:
Do you still need this? I know this pr was opened a long time ago but i don't think this should be required anymore.